x-pack/platform/plugins/shared/ai_infra/product_doc_base/public/services/installation/installation_service.ts

```diff
@@ -22,19 +23,40 @@ export class InstallationService {
     this.http = http;
   }
 
-  async getInstallationStatus(): Promise<InstallationStatusResponse> {
-    return await this.http.get<InstallationStatusResponse>(INSTALLATION_STATUS_API_PATH);
+  async getInstallationStatus(
+    params: { inferenceId?: string } = {}
+  ): Promise<InstallationStatusResponse> {
+    const inferenceId = params?.inferenceId ?? defaultInferenceEndpoints.ELSER;
+
+    const response = await this.http.get<InstallationStatusResponse>(INSTALLATION_STATUS_API_PATH, {
+      query: { inferenceId },
+    });
+
+    return response;
   }
 
-  async install(): Promise<PerformInstallResponse> {
-    const response = await this.http.post<PerformInstallResponse>(INSTALL_ALL_API_PATH);
+  async install(params: { inferenceId?: string } = {}): Promise<PerformInstallResponse> {
+    const inferenceId = params?.inferenceId ?? defaultInferenceEndpoints.ELSER;
+
+    const response = await this.http.post<PerformInstallResponse>(INSTALL_ALL_API_PATH, {
+      body: JSON.stringify({ inferenceId }),
+    });
+
     if (!response.installed) {
-      throw new Error('Installation did not complete successfully');
+      throw new Error(
+        `Installation did not complete successfully.\n${response.failureReason ?? ''}`
```

Comment:
I think customers will find this error quite confusing:

<img width="295" alt="image" src="https://github.com/user-attachments/assets/9990b3dd-a09d-4c7d-babe-4160bdfd2adf" />


This is particularly cryptic:

> End of central directory record signature not found. either not a zip file or file is truncated

What action are users expected to take from seeing this? And under what circumstances does it happen?